modules/keyless-authentication-robot-accounts.adoc

```diff
@@ -0,0 +1,268 @@
+:_content-type: PROCEDURE
+[id="keyless-authentication-robot-accounts"]
+= Keyless authentication with robot accounts
+
+In previous versions of {productname}, robot account tokens were valid for the lifetime of the token unless deleted or regenerated. Tokens that do not expire have security implications for users who do not want to store long-term passwords or manage the deletion, or regeneration, or new authentication tokens. 
+
+With {productname} {producty}, {productname} administrators are provided the ability to exchange {productname} robot account tokens for an external OIDC token with either Red Hat SSO Key Cloak or Microsoft Entra ID. This allows robot accounts to leverage short-lived, or _ephemeral tokens_, that last one hour. Ephemeral tokens are refreshed regularly and can be used to authenticate individual transactions. 
```

Comment:
it's the opposite. Exchange external OIDC tokens for a short lived robot token

Reply:
fixed ty